modules/ROOT/pages/connector-archetype.adoc

```diff
@@ -39,8 +40,8 @@ You'll then have to specify interactively the properties of your project:
 * *version:* the version of your connector _(default value: 1.0-SNAPSHOT)_
 Follow the http://maven.apache.org/guides/mini/guide-naming-conventions.html[maven naming convention guide]
 * *package* the package in which the connector source files will be created _(default value: the group id of the connector)_
-* *bonitaVersion:* the targeted Bonita version
- ** A Bonita connector project depends on _org.bonitasoft.engine:bonita-common_. To avoid potential conflicts / errors at runtime, you should use the Bonita version of your runtime environment.
+* *bonitaVersion:* the xref:product-versioning#_technical_id[technical id] of the targeted Bonita version
+ ** A Bonita connector project depends on _org.bonitasoft.engine:bonita-common_. To avoid potential conflicts / errors at runtime, you should use the technical id of the Bonita version of your runtime environment.
```

Comment:
```suggestion
 ** A Bonita connector project depends on _org.bonitasoft.engine:bonita-common_. To avoid potential conflicts / errors at runtime, you should use the technical id of the Bonita version of your runtime environment (7.13.0 for Bonita 2021.2)
```